recipes-bsp/u-boot/files/0013-iot2050-config-the-setexpr-cmd.patch

```diff
@@ -0,0 +1,27 @@
+From 3fe2e1c82a026f5282bcddcd49087627c87e6956 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Tue, 8 Mar 2022 13:42:34 +0800
+Subject: [PATCH 13/14] iot2050: config the setexpr cmd
+
+enable the setexpr cmd under u-boot.
+
+Signed-off-by: chao zeng <chao.zeng@siemens.com>
+---
+ configs/iot2050_pg2_defconfig | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/configs/iot2050_pg2_defconfig b/configs/iot2050_pg2_defconfig
+index 83866af4c6..1d979ac6b4 100644
+--- a/configs/iot2050_pg2_defconfig
++++ b/configs/iot2050_pg2_defconfig
```

Comment:
Will the code in u-boot-script not pass along the setexpr command when running on PG1?

Reply:
> Will the code in u-boot-script not pass along the setexpr command when running on PG1?
currently only m.2 need load the dtbo, the command setexpr PG1 not required